docs/decommission.rst

```diff
@@ -0,0 +1,80 @@
+#. **Disconnect the firewall and the servers from the internet.**
+   Be sure to inform your network administrator of any changes to devices on
+   your network.
+#. **Wipe or destroy the USB drives.**
+   Because the USB drives used for SecureDrop are all LUKS-encrypted,
+   reformatting the USB drives (in particular, overwriting a portion of internal
+   storage called the **LUKS header**) should be sufficient to make any existing
+   data on those drives unrecoverable.
+
+   For example, you could use your *primary Tails USB* to launch Gnome Disks,
+   insert and identify the USB drive you are trying to erase, and reformat this
+   drive with a new, LUKS-encrypted partition, erasing the existing partition
+   data.
+
+   .. caution:: Be **very** sure you are reformatting the right drive.
+      You may want to use the Secure Viewing Station laptop for this procedure
+      to reduce the risk of accidentally erasing a drive on your regular-use
+      machine.
+
+   You may also choose to destroy the drives by physical means, such as using a
+   hammer or purpose-built shredder to pulverize or destroy the drive.
+#. **Wipe or destroy the storage drives on the servers.**
+   SecureDrop submissions are stored GPG-encrypted on the Application server.
+   Unless your SecureDrop Application Key is compromised (or a significant
+   vulnerability in GPG is discovered), access to the servers does not guarantee
+   access to the submissions and messages you have received.
+
+   That said, there is still some sensitive information on the servers,
+   including system logs and the SecureDrop database, which would yield
+   information on the number of submissions and replies stored on the server.
+
+   Physically destroying or encrypting the storage drives on the servers are the
+   best ways to ensure that data on the drives cannot be recovered.
+
+   Physically destroying SSD drives is not as straightforward as destroying
+   older hard drives, and requires some research. Depending on your threat
+   model, you may choose instead to write over the information on the existing
+   drives. One way to do this is to re-install a clean version of Ubuntu server
+   with full-disk encryption enabled. During the disk partitioning portion of
+   the installation wizard, select *Guided - use entire disk and set up
+   encrypted LVM*.
+#. **Destroy other Transfer or Export media, if applicable.**
+#. **Factory-reset the firewall (optional).**
+#. **Notify the SecureDrop Support team that your instance is no longer active.**
+   If you have any questions about the decommissioning process, or about other
+   secure communications options, please feel free to contact us at
+   securedrop@freedom.press
+   (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__)
+   or via the `support portal <https://support.freedom.press/>`__.
+
+.. _backup guidelines: https://docs.securedrop.org/en/master/backup_and_restore.html#backing-up
```

Comment:
To ensure stability of links, it's best to avoid using absolute URLs for references within the documentation itself; you can use a `:doc:` type link to refer to the whole doc, or insert a named reference in the target document near a specific anchor, and then use a `:ref:` link.

Reply:
fixed in f38127f